packages/runtime/container-runtime/src/runningSummarizer.ts

```diff
@@ -108,12 +109,17 @@ export class RunningSummarizer implements IDisposable {
         private readonly stopSummarizerCallback: (reason: SummarizerStopReason) => void,
         { disableHeuristics = false }: Readonly<Partial<ISummarizerOptions>> = {},
     ) {
+        const telemetryProps: ISummarizeRunnerTelemetry = {
+            summarizeCount: () => this.summarizeCount,
+            summarizerSuccessfulAttempts: () => this.totalSuccessfulAttempts,
+        };
+
         this.logger = ChildLogger.create(
             baseLogger, "Running",
             {
                 all: {
-                    summarizeCount: () => this.summarizeCount,
```

Comment:
why not use all: telemetryProps ?

Reply:
Good idea, I had to extend the change the interface ISummarizeRunnerTelemetry to extend ITelemetryLoggerPropertyBag

Otherwise, it was complaining.